client/.yarnrc.yml

```diff
@@ -0,0 +1 @@
+yarnPath: .yarn/releases/yarn-3.6.3.cjs
```

Comment:
If you want to use `node_modules`, you have to add: `nodeLinker: node-modules`

Reply:
and now you know why I asked for a yarn3 intro chat